src/quantum_annealing/dwave_solvers.rs

```diff
@@ -0,0 +1,29 @@
+pub enum DWaveSolver {
+    Advantage5_1,
+    DW_2000Q
+}
+
+pub get_solver_qubits(solver: DWaveSolver) -> Vec<i32>{
```

Comment:
We could add a very brief comment explaining where these constants came from or how they were generated. Doesn't have to be part of this PR, but eventually we want people to be able to reproduce this file if needed I think.